GNUmakefile

```diff
@@ -9,6 +9,10 @@ PROVIDER_VERSION = 99.99.99
 PLUGINS_PATH = ~/.terraform.d/plugins
 PLUGINS_PROVIDER_PATH=$(PROVIDER_HOSTNAME)/$(PROVIDER_NAMESPACE)/$(PROVIDER_TYPE)/$(PROVIDER_VERSION)/$(PROVIDER_TARGET)
 
+# Use a parallelism of 3 by default for tests, overriding whatever GOMAXPROCS is set to.
+TEST_PARALLELISM?=3
+TESTARGS?=-short
```

Comment:
Do you think it might be worth leaving `testacc` as the default slow mega test and having a `test-ci` target for the short mode?

Reply:
Keeping the default with -short that can be overridden when executing individual tests.